index.js

```diff
@@ -69,17 +69,17 @@ const createClient = (profile, request = _request) => {
 			if (!isNonEmptyString(opt.earlierThan)) {
 				throw new Error('opt.earlierThan must be a non-empty string.')
 			}
-			if ('when' in opt) {
-				throw new Error('opt.earlierThan and opt.when are mutually exclusive.')
+			if (('departure' in opt) || ('arrival' in opt)) {
```

Comment:
shouldn't this be `&&` instead of `||`?

Reply:
We want to check if `earlierThan`/`laterThan` and `departure`/`arrival` are being used together. The code is somewhat inelegant but – afaict – does what is supposed to.

Reply:
Sorry, I didn't read correctly. What I mistook this for is: Shouldn't `departure` and `arrival` be mutually exclusive as well?